teamsite/main/templates/main/home.html

```diff
@@ -1,24 +1,306 @@
-<!doctype html>
-<html lang="en">
+{% load static %}
+<!DOCTYPE html>
+<html>
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Home</title>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>online course website</title>
+
+	<!---custom css link---->
+	<link rel="stylesheet" href="{% static 'main/css/final.css' %}">
+	<link rel="stylesheet" href="https://unpkg.com/boxicons@latest/css/boxicons.min.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;700;800;900&display=swap" rel="stylesheet">
 </head>
 <body>
-<ul>
-    <li>
-        User: {{ user }}
-    </li>
-    <li>
-        {% if user.is_authenticated %}
-            <a href="{% url 'logout' %}">Logout</a>
-        {% else %}
-            <a href="{% url 'login' %}">Login</a>
-        {% endif %}
-    </li>
-</ul>
+
+	<!---start header---->
+	<header>
+		<a href="#" class="logo">
+			<img src="{% static 'main/img/logo-no-background.png' %}">
+		</a>
+
+		<ul class="navbar">
+			<li><a href="#home">Home</a></li>
+			<li><a href="#tests">Tests</a></li>
+			<li><a href="#courses">Courses</a></li>
+			<li><a href="#about">Forum</a></li>
+			<li><a href="#contact">Contact</a></li>
+		</ul>
+
+		<div class="header-icons">
+			<!---<a href="#"><i class='bx bx-user'></i></a>
+			<a href="#"><i class='bx bx-heart' ></i></a>
+			<a href="#"><i class='bx bx-cart' ></i></a>--->
+			<div class="bx bx-menu" id="menu-icon"></div>
+		</div>
+	</header>
+
+	<!---start home section---->
+	<section class="home" id="home">
+		<div class="home-text">
+			<h6>Best online learning platform</h6>
+			<h1>Accessible Online Courses For All</h1>
+			<p>Own your future learning new skills online</p>
+			<div class="latter">
+				<a href="#" class="btn">Get Started</a>
+			</div>
+		</div>
+
+		<div class="home-img">
+			<img src="{% static 'main/img/reshot-illustration-teamwork-collaboration-Z2FRC687LE.png' %}">
+		</div>
+	</section>
+
+	<!---start container section---->
+	<section class="container">
+		<div class="container-box">
```

Comment:
Are the next container blocks are hardcoded?

Reply:
Yeah